### PR TITLE
add radtraq

### PR DESCRIPTION
**Pull request by akrherz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
-  Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/radtraq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
```
Comment (mgrover1) [inline recipes/radtraq/meta.yaml:22]:
```suggestion
    - pip
    - setuptools
```

File: recipes/radtraq/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  run:
+    - python >=3.8
+    - pandas <2.0
+    - act-atmos
+    - numpy
+    - scipy
+    - xarray
+    - matplotlib-base
+    - netcdf4
+    - pint
```
Comment (mgrover1) [inline recipes/radtraq/meta.yaml:33]:
```suggestion
    - pint
    - setuptools
```

**Comment by mgrover1** [comment]
Since it is used in the setup.py file we need it there too

File: recipes/radtraq/meta.yaml (hunk 2 above)
Comment (akrherz) [reply recipes/radtraq/meta.yaml:33]: Erm, I believe the error is from using an old version of versioneer unsupported by python 3.11 ???

**Comment by akrherz** [comment]
awaiting upstream 0.0.8 pypi release and will update this PR.

**Comment by akrherz** [comment]
@conda-forge/help-python, ready for review

**Comment by akrherz** [comment]
A kind ping to @dopplershift to see if he can review / push this PR over the finish line.  TIA

**Comment by akrherz** [comment]
kind ping to @ocefpaf for a review / merge, thanks

**Comment by ocefpaf** [comment]
@akrherz did you get all the maintainers to acknowledge they are willing to maintain this recipe?

**Comment by mgrover1** [comment]
I am willing to maintain the recipe

**Comment by akrherz** [comment]
@ocefpaf I wasn't sure of the mechanics of that, do they just need to approve the PR?  Anyway, [this comment](https://github.com/ARM-Development/RadTraQ/pull/47#issuecomment-1875794198) perhaps suffices?  sorry for the ignorance :(

**Comment by ocefpaf** [comment]
They can comment here, like @mgrover1 did above.

**Comment by zssherman** [comment]
@akrherz @ocefpaf I am willing to maintain this recipe as well!

**Comment by AdamTheisen** [comment]
@akrherz @ocefpaf I am willing to maintain this recipe

**Comment by mgrover1** [comment]
@ocefpaf that should be everyone :)

**Pull request merged**